{{cookiecutter.project_slug}}/local.yml

```diff
@@ -13,6 +13,7 @@ services:
     container_name: django
     depends_on:
       - postgres
+      - redis
```

Comment:
This is only necessary if celery is installed

Reply:
Agree, there is no redis container if Celery isn't used.